packages/ses/docs/lockdown.md

```diff
@@ -369,16 +369,37 @@ the container to exit explicitly, and we highly recommend setting
 
 ## `evalTaming` Options
 
-**Background**: In some environment, development mode allows `unsafe-eval` but the production mode does not.
-This option allows disabling the replacement of `eval` and `Function` so that things like `eval-source-map` works in development mode.
-Assume no `eval-source-map` (or things like that) is used in production, and no `eval` is allowed (e.g. by Content Security Policy) in production, taming `eval` and `Function` constructor actally hurts developer experience.
-
-This option should ONLY be used when you have situation like described above or you think you have reasonable use case to do so.
-Security audit is NOT performed on this option and you SHOULD NEVER use it in production.
-
-Strongly suggest to use [Trusted Types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/trusted-types) when you have use cases like this.
-
-This option only applies to the start compartment. Child compartment will always use value `"safeEval"`.
+**Background**: Every compartment including the implied compartment in the
+initial JavaScript realm, there are evaluators `eval` and `Function`.
+The default lockdown behavior isolates all of these evaluators.
+
+Replacing the realm's initial evaluators is not necessary to ensure the
+isolation of guest code because guest code must not run in the initial realm.
+However, replacing these evaluators is the safest and therefore default
+evaluator taming option (`"safe"`) because it may limit the harm that guest code can cause
+if it escapes its assigned compartment.
+
+However, only the exact `eval` function from the initial realm can be used to
+perform direct-eval, which runs in the caller's scope (dynamic scope).
```

Comment:
```suggestion
perform direct-eval, which runs in the lexical scope in which the direct-eval syntax appears (direct-eval is a special form rather than a function call).
```
I can see the rationale for calling this dynamic scope, but I think it is better to think of direct-eval as a special form, like `if` is, and therefore naturally executes in the lexical scope of the code it appears in. Unfortunately, I doubt the term "special form" is well known outside Scheme. Better terminology would be good, but go with "special form" if we don't come up with anything better.

Or, frankly, you could just omit the parenthetical. It's ok if this document isn't the one to try to explain anything more about direct-eval.
```suggestion
perform direct-eval, which runs in the lexical scope in which the direct-eval syntax appears.
```